_posts/2024-04-13-firo-tokenomics-and-halving.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Firo Tokenomics Discussion with upcoming Halving"
+summary: "Firo's Halving in mid September 2024"
+tags: [community, dev]
+author: "Reuben Yap"
+img: "/blog/assets/halvingtokenomics2024/banner.png"
+---
+With the upcoming supply halving in mid September of this year, many community members have raised whether the block reward distribution or tokenomics of Firo will be revisited. The last time this was done was in [May 2022](https://forum.firo.org/t/tokenomics-and-funding-division-of-block-reward-discussion-and-community-matching-fund/2061).
+
+Link to the 2024 Firo Tokenomics Discussion with upcoming Halving thread is located [here](https://forum.firo.org/t/firo-tokenomics-discussion-with-upcoming-halving/3162).
+
+Currently it is divided as follows:
+
+* 50% Masternodes 
+* 25% Miners 
+* 15% Dev reward 
+* 10% Community Fund 

```

Comment:
Would be better to have each  point on its on line, but it's good to go as it is.

Reply:
ok will do next time